magento/magento2-page-builder
# magento/magento2-page-builder#525: A Slider element is not responsive on the mobile view in the Firefox browser.

### Description

This pull request will add a fix for the responsive behavior of the Column element in Firefox on the mobile view.

Added ` max-width: 100%;` to `.pagebuilder-column {...}`

### Fixed Issue

[#525](https://github.com/magento/magento2-page-builder/issues/525) A Slider element is not responsive on the mobile view in the Firefox browser.


### Manual testing scenarios

1. Create a CMS page
2. Add Column element to a Row
3. Add Slider to any column /slider styles are optional/
3. Save page
4. Go to a page on storefront
5. Check page in mobile view


### Checklist
 [+] Pull request has a meaningful description of its purpose
 [+] All commits are accompanied by meaningful commit messages
 [ ] All new or changed code is covered with unit/integration tests (if applicable)
 [ ] All automated tests passed successfully (all builds are green)
